docs/BrowserObject.md

```diff
@@ -82,14 +83,51 @@ console.log(browser.config)
         connectionRetryTimeout: 120000,
         connectionRetryCount: 3,
         specs: [ 'err.js' ],
-        fakeUser: 'maxmustermann', // <-- custom option
-        fakePassword: 'foobar', // <-- custom option
+        fakeUser: 'maxmustermann', // <-- custom configuration
+        fakePassword: 'foobar', // <-- custom configuration
+        baseUrl: 'example.com', // <-- custom configuration
         // ...
  */
 
 console.log(browser.config.fakeUser) // outputs: "maxmustermann"
 ```
 
+### Configurations versus Options
+
+Custom configurations should not be confused with [Options](Options.md), which are accessed separately.
+
+```js
+console.log(browser.options)
+/**
+ * outputs:
+ * {
+        protocol: 'http',
+        port: 4444,
+        hostname: 'localhost',
+        baseUrl: 'example.com',
+        // ...
+ */
+```
+
+When using the WDIO testrunner, if any configuration and option keys conflict in name (e.g. `baseUrl` in the following code snippets) the option value will take precedence and overwrite the config value.
+
+```js
+// wdio.conf.js
+exports.config = {
+    baseUrl: 'example.com'
+}
+```
+
+```bash
+# testrunner invocation
+$ npm wdio wdio.conf.js --baseUrl foobar.com
+```
+
+```js
+console.log(browser.config.baseUrl) // 'foobar.com'
+console.log(browser.options.baseUrl) // 'foobar.com'
```

Comment:
shouldn't these values be different given that the one "will take precedence and overwrite the config value"?

Reply:
This is the result of that behavior.

In the previous two snippets, I set one to `example.com` and one to `foobar.com`. But both resolve to `foobar.com` during test execution.

Reply:
> Maybe I should make that more clear.

My initial question was, how would I get the value that was passed in as wdio parameter